src/System.Private.CoreLib/src/System/Reflection/AssemblyName.CoreRT.cs

```diff
@@ -0,0 +1,27 @@
+// Licensed to the .NET Foundation under one or more agreements.
+// The .NET Foundation licenses this file to you under the MIT license.
+// See the LICENSE file in the project root for more information.
+
+using System.Runtime.Serialization;
+
+namespace System.Reflection
+{
+    public sealed partial class AssemblyName : ICloneable, IDeserializationCallback, ISerializable
+    {
+        public void nInit()
+        {
+            RuntimeAssemblyName runtimeAssemblyName = AssemblyNameParser.Parse(_name);
+            runtimeAssemblyName.CopyToAssemblyName(this);
+        }
+
+        private byte[] nGetPublicKeyToken()
```

Comment:
It would better to call this `ComputePublicKeyToken` and rename it in CoreCLR as necessary.